parsers/test/mocks/ERCOT.json

```diff
@@ -0,0 +1,16231 @@
+          "Other": {
+            "gen": 0.1,
+            "hsl": 0,
+            "seasonalCapacity": 174
+          },
+          "Power Storage": {
+            "gen": 406.4,
+            "hsl": 5739.7,
+            "seasonalCapacity": 9514
+          },
+          "Solar": {
+            "gen": 0.8,
+            "hsl": 0.6,
+            "seasonalCapacity": 27167
+          },
+          "Wind": {
+            "gen": 22046.1,
+            "hsl": 22376,
+            "seasonalCapacity": 39533
+          },
+          "Natural Gas": {
+            "gen": 10122.5,
+            "hsl": 13784.6,
+            "seasonalCapacity": 68234
+          }
+        },
+        "2024-11-23 00:14:57-0600": {
+          "Coal and Lignite": {
+            "gen": 5122.7,
+            "hsl": 9881.4,
+            "seasonalCapacity": 14713
+          },
+          "Hydro": {
+            "gen": 5.8,
+            "hsl": 6,
+            "seasonalCapacity": 575
+          },
+          "Nuclear": {
+            "gen": 5085.9,
+            "hsl": 5098.2,
+            "seasonalCapacity": 5268
+          },
+          "Other": {
+            "gen": 0.1,
+            "hsl": 0,
+            "seasonalCapacity": 174
+          },
+          "Power Storage": {
+            "gen": 244.6,
+            "hsl": 4044.3,
+            "seasonalCapacity": 9514
+          },
+          "Solar": {
+            "gen": 1.1,
+            "hsl": 0.9,
+            "seasonalCapacity": 27167
+          },
+          "Wind": {
+            "gen": 22105.2,
+            "hsl": 22420.5,
+            "seasonalCapacity": 39533
+          },
+          "Natural Gas": {
+            "gen": 10032.9,
+            "hsl": 13467.1,
+            "seasonalCapacity": 68234
+          }
+        },
+        "2024-11-23 00:19:57-0600": {
+          "Coal and Lignite": {
+            "gen": 5137.6,
+            "hsl": 9881.4,
+            "seasonalCapacity": 14713
+          },
+          "Hydro": {
+            "gen": 5.7,
+            "hsl": 6,
+            "seasonalCapacity": 575
+          },
+          "Nuclear": {
+            "gen": 5084.9,
+            "hsl": 5097.8,
+            "seasonalCapacity": 5268
+          },
+          "Other": {
+            "gen": 0.1,
+            "hsl": 0,
+            "seasonalCapacity": 174
+          },
+          "Power Storage": {
+            "gen": 216.8,
+            "hsl": 2876.6,
+            "seasonalCapacity": 9514
+          },
+          "Solar": {
+            "gen": 1.1,
+            "hsl": 0.9,
+            "seasonalCapacity": 27167
+          },
+          "Wind": {
+            "gen": 21951.8,
+            "hsl": 22343.6,
+            "seasonalCapacity": 39533
+          },
+          "Natural Gas": {
+            "gen": 9955.2,
+            "hsl": 13310.1,
+            "seasonalCapacity": 68234
+          }
+        },
+        "2024-11-23 00:24:57-0600": {
+          "Coal and Lignite": {
+            "gen": 5063.9,
+            "hsl": 9881.4,
```

Comment:
Do you know what hsl stands for?

If it's the capacity that would be real interesting as technically we can parse live capacities. (follow up pr if that's the case)

Reply:
Maybe hourly system load (google search). But I cannot find the documentation for the API so I'm not sure.